### PR TITLE
fix: slider styles and accessibility

### PR DESCRIPTION
**Pull request by NadiaPosch**

### CODE REVIEW

File: src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -19,49 +24,74 @@ export type SliderProps = {
...
     return (
         <AnimateSharedLayout>
             <ul
                 data-test-id="slider"
                 className="w-full grid grid-flow-col auto-cols-fr justify-evenly p-0 border border-black-20 m-0 bg-black-0 rounded font-sans text-s list-none"
+                {...radioGroupProps}
             >
                 {items.map((item) => {
                     const isActive = item.id === activeItemId;
+                    const radioProps = {
```
Comment (dwirz) [inline src/components/Slider/Slider.tsx:53]: inline since only used once?

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -19,49 +24,74 @@ export type SliderProps = {
...
+                        value: item.id,
+                        "aria-label": isIconItem(item) ? item.ariaLabel : item.name,
```
Comment (dwirz) [inline src/components/Slider/Slider.tsx:55]: since `item.ariaLabel` is optional should we use a fallback like `item ${index}`?

Comment (NadiaPosch) [reply src/components/Slider/Slider.tsx:55]: For icon items it's not optional (only for the slider component) and for text items the name always has to be set as well, so there shouldn't be a case where a fallback is necessary.

File: package.json [hunk 3]
```diff
@@ -107,6 +107,10 @@
         "react-router-dom": "^5.2.0"
     },
     "dependencies": {
+        "@react-aria/focus": "^3.4.0",
+        "@react-aria/radio": "^3.1.4",
+        "@react-aria/visually-hidden": "^3.2.2",
+        "@react-stately/radio": "^3.3.1",
```
Comment (SamuelAlev) [inline package.json:113]: You can install [react-aria](https://www.npmjs.com/package/react-aria) instead

**Review by dwirz — APPROVED** [review]

**Pull request merged**